api/routes/settings.ts

```diff
@@ -0,0 +1,63 @@
+import type { FastifyPluginCallbackTypebox } from '@fastify/type-provider-typebox';
+import { Type } from '@sinclair/typebox';
+
+export const settingRoutes: FastifyPluginCallbackTypebox = (
+  fastify,
+  _options,
+  done
+) => {
+  fastify.put(
+    '/update-my-profileui',
+    {
+      schema: {
+        body: Type.Object({
+          profileUI: Type.Object({
+            isLocked: Type.Boolean(),
+            showAbout: Type.Boolean(),
+            showCerts: Type.Boolean(),
+            showDonation: Type.Boolean(),
+            showHeatMap: Type.Boolean(),
+            showLocation: Type.Boolean(),
+            showName: Type.Boolean(),
+            showPoints: Type.Boolean(),
+            showPortfolio: Type.Boolean(),
+            showTimeLine: Type.Boolean()
+          })
+        }),
+        200: Type.Object({
+          message: Type.String(),
+          type: Type.Literal('success')
+        }),
+        500: Type.Object({
+          message: Type.String(),
+          type: Type.Literal('danger')
+        })
```

Comment:
```suggestion
        response: {
          200: Type.Object({
            message: Type.String(),
            type: Type.Literal('success')
          }),
          500: Type.Object({
            message: Type.String(),
            type: Type.Literal('danger')
          })
        }
```

The response types needs to be put in a `response` object otherwise TypeBox won't pick up on the type and enforce it.